aiocassandra.py

```diff
@@ -1,10 +1,62 @@
 import asyncio
+from collections import deque
 from functools import partial
 from types import MethodType
 
+from async_generator import async_generator, yield_
 from cassandra.cluster import Session
 
-__version__ = '1.1.1a'
+__version__ = '1.2.0a0'
```

Comment:
Dropping env support deserves a major: according to semver it's a backwards incompatible change.